packages/console/src/ConsoleInput.scss

```
@@ -74,3 +74,16 @@ $console-input-margin-color: $foreground;
     }
   }
 }
+
+.docs {
+  min-width: 200px;
+  
+  hr {
+    background-color: $text-muted;
+    height: 1px;
+  }
+
+  blockquote {
+    margin-left: 20px;
```

```suggestion
    margin-left: $spacer-3;
```

Or spacer-2 whichever is closer.